src/directives/mobilegeolocation.js

```diff
@@ -21,6 +21,9 @@ goog.require('ol.geom.Point');
  *        ngeo-mobile-geolocation-options="ctrl.mobileGeolocationOptions">
  *      </button>
  *
+ * @htmlAttribute {???} ngeo-mobile-geolocation ???
```

Comment:
Or in the example that `ngeo-mobile-geolocation=""` should be `ngeo-mobile-geolocation`?